Stackable/ScrollingStackView.swift

```diff
@@ -7,6 +7,7 @@
 //
 
 import Foundation
+import UIKit
```

Comment:
Nitpicky, but can you kill the `Foundation` imports, since `UIKit` contains it

Reply:
👍